src/tokenizer.ts

```
@@ -0,0 +1,585 @@
+ * @param tokens The tokens generated from the chunk.
+ */
+export function transitionFromLiteralToEscapeMode(
+  state: LiteralMode,
+  tokens: Token[],
+) {
+  flushBuffer(state, tokens, TokenType.Literal);
+
+  const castedState = state as unknown as EscapeMode;
+  castedState.type = Mode.Escape;
+
+  delete (state as AnyMode).buffer;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)